### PR TITLE
Update migrations to reflect changes

### PR DESCRIPTION
**Pull request by rain-on**

### CODE REVIEW

**Comment by jframe** [comment]
These migrations fail for me with an existing DB from the 0.2 release with the error "ERROR: Validate failed: Migrations have failed validation"

Also don't think having these migrations are a good idea or needed. The DB did change on the master builds, but slashing protection was a hidden option and disabled so don't think that is an issue.

File: slashing-protection/src/main/resources/migrations/postgresql/V1__initial.sql [hunk 1]
```diff
@@ -6,15 +6,15 @@ CREATE TABLE validators (
 CREATE TABLE signed_blocks (
     validator_id INTEGER NOT NULL,
     slot NUMERIC(20) NOT NULL,
-    signing_root BYTEA,
+    signing_root BYTEA NOT NULL,
```
Comment (jframe) [inline slashing-protection/src/main/resources/migrations/postgresql/V1__initial.sql:9]: We shouldn't change what V1 was as there will be existing databases using this schema

File: slashing-protection/src/main/resources/migrations/postgresql/V2__removeNullConstraint.sql [hunk 2]
```diff
@@ -0,0 +1,5 @@
+ALTER TABLE signed_blocks
```
Comment (jframe) [inline slashing-protection/src/main/resources/migrations/postgresql/V2__removeNullConstraint.sql:1]: Similarly should just get rid of this, these changes are part of the V1 that was in the 0.2 release so think it's fine that they stay that way.

File: slashing-protection/src/main/resources/migrations/postgresql/schema [hunk 3]
```diff
@@ -0,0 +1,177 @@
+--
```
Comment (jframe) [inline slashing-protection/src/main/resources/migrations/postgresql/schema:1]: Can we remove this. As it is I don't think it adds value. It contains a lot unnecessary changes such as the flyway database and ownership permissions.

File: slashing-protection/src/main/resources/migrations/postgresql/V1__initial.sql (hunk 1 above)
Comment (rain-on) [reply slashing-protection/src/main/resources/migrations/postgresql/V1__initial.sql:9]: deal

File: slashing-protection/src/main/resources/migrations/postgresql/V2__removeNullConstraint.sql (hunk 2 above)
Comment (rain-on) [reply slashing-protection/src/main/resources/migrations/postgresql/V2__removeNullConstraint.sql]: deal

File: slashing-protection/src/main/resources/migrations/postgresql/schema (hunk 3 above)
Comment (rain-on) [reply slashing-protection/src/main/resources/migrations/postgresql/schema]: gone

**Review by jframe — APPROVED** [review]

File: slashing-protection/src/main/resources/migrations/postgresql/V2__removeUniqueConstraints.sql [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+  WHERE conrelid = 'signed_blocks'::regclass
+  AND contype = 'u');
+
+  _signed_attestation_conn text := (
+  SELECT conname
+  FROM pg_constraint
+  WHERE conrelid = 'signed_attestations'::regclass
+  AND contype = 'u');
+BEGIN
+  EXECUTE 'ALTER TABLE signed_blocks DROP CONSTRAINT ' || _signed_block_conn;
```
Comment (jframe) [inline slashing-protection/src/main/resources/migrations/postgresql/V2__removeUniqueConstraints.sql:16]: Does removing these constraints affect performance at all? Essentially this was acting a primary key before
